Cargo.toml

```diff
@@ -100,7 +100,7 @@ massa_wallet = {path = "./massa-wallet"}
 
 # Massa projects dependencies
 massa-proto-rs = {git = "https://github.com/massalabs/massa-proto-rs", "rev" = "18ec02f"}
-massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "branch" = "main"}
+massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "branch" = "evm-update"}
```

Comment:
To update when merging :)

Reply:
Yep was just waiting for the runtime side to be merged first